### PR TITLE
Make it clear that the bundle is deprecated in the README file

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,9 @@ AsseticBundle
 =============
 
 The `AsseticBundle` provides integration of the [Assetic](https://github.com/kriswallsmith/assetic)
-library into the Symfony2 framework.
+library into the Symfony framework (up to Symfony 3.4).
+
+As of Symfony 4.0, Symfony applications should use Webpack Encore.
```
Comment (xabbuh) [inline README.md]: Maybe prefix this with `**CAUTION**:` to put some emphasize on this paragraph?

Comment (fabpot) [reply README.md]: Good idea, done.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,9 @@ AsseticBundle
 =============
 
 The `AsseticBundle` provides integration of the [Assetic](https://github.com/kriswallsmith/assetic)
-library into the Symfony2 framework.
+library into the Symfony framework (up to Symfony 3.4).
+
+**CAUTION**: As of Symfony 4.0, Symfony applications should use Webpack Encore.
```
Comment (xabbuh) [inline README.md]: Sorry, I should have commented that with the other remark: Should we also link "Webpack Encore" with https://symfony.com/doc/current/frontend.html?

Comment (fabpot) [reply README.md]: Done :)

**Review by xabbuh — APPROVED** [review]

**Pull request merged**